scripts/output.js

```diff
@@ -233,17 +233,29 @@ function downloadCSV(csvContent, old) {
 	create_zip(csvContent, old);
 }
 
+/*
+*This function downloads the image into the designated folder when given the uri and name
+*/
 function downloadURI(uri, name) {
-	var safeName = name;
-	var safeUri = uri.slice(22);
-	console.log("in image", safeUri);
-	var imgObj = {
-		name:safeName+".png",
-		uri: safeUri
-	};
-	console.log("imgobj", imgObj);
-	imgList.push(imgObj);
-
+    try {
+        //checks to see if the uri or name is null
+        if (uri === null || name === null) throw "The uri or name for your image is null.";
```

Comment:
Is the idea that if "Exit" is clicked this error will be logged? (For me, I can't get this error to show up)

Reply:
It's just error checking to make sure the URI and name isn't null lol, it should only show the error when there is an error (which there shouldn't be)

Reply:
Lol okay